lib/Runtime/Library/Chakra.Runtime.Library.vcxproj

```diff
@@ -157,11 +157,14 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)IntlEngineInterfaceExtensionObject.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)JavascriptErrorDebug.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)JavascriptListIterator.cpp" />
+    <ClCompile Include="AtomicsObject.cpp" />
+    <ClCompile Include="SharedArrayBuffer.cpp" />
```

Comment:
will do.

---
In reply to: [77406543](https://github.com/Microsoft/ChakraCore/pull/1533#discussion_r77406543) [](ancestors = 77406543)